stetho/src/main/java/com/facebook/stetho/inspector/network/AsyncPrettyPrinterExecutorHolder.java

```diff
@@ -16,5 +16,21 @@
  * A holder class for the executor service used for pretty printing related tasks
  */
 class AsyncPrettyPrinterExecutorHolder {
-  public static final ExecutorService sExecutorService = Executors.newCachedThreadPool();
+  private static ExecutorService sExecutorService;
+
+  public static void init() {
```

Comment:
This shouldn't be public. I would also name it something like `ensureInitialized`